### PR TITLE
Unit tests updated

### PR DESCRIPTION
**Pull request by awohsen**
The description is as below comments:

### CODE REVIEW

File: tests/QueryBuilderTest/HelpersTest/EscapeTest.php
Comment (awohsen) [inline tests/QueryBuilderTest/HelpersTest/EscapeTest.php:1]: Based on:
https://github.com/SarafApp/ReactPHP-MySQL-QueryBuilder/blob/170c7d3076ade1e8cf8f65a51a61b432cd07de43/src/QueryBuilder/Helpers/Escape.php#L53

File: tests/QueryBuilderTest/ClausesTest/SelectTest.php
Comment (awohsen) [inline tests/QueryBuilderTest/ClausesTest/SelectTest.php:1]: Based on feature:
https://github.com/SarafApp/ReactPHP-MySQL-QueryBuilder/blob/170c7d3076ade1e8cf8f65a51a61b432cd07de43/src/QueryBuilder/Clauses/Select.php#L209

File: tests/QueryBuilderTest/CapabilityTest/WhereTest.php
Comment (awohsen) [inline tests/QueryBuilderTest/CapabilityTest/WhereTest.php:1]: Based on:
https://github.com/SarafApp/ReactPHP-MySQL-QueryBuilder/blob/170c7d3076ade1e8cf8f65a51a61b432cd07de43/src/QueryBuilder/Helpers/Escape.php#L53

**Review by hasanparasteh — APPROVED** [review]
LGTM

**Pull request merged**